pandas/tests/arithmetic/test_numeric.py

```diff
@@ -127,7 +128,12 @@ def test_numeric_cmp_string_numexpr_path(self, box_with_array):
         result = obj != "a"
         tm.assert_equal(result, ~expected)
 
-        msg = "Invalid comparison between dtype=float64 and str"
+        msg = "|".join(
+            [
+                "Invalid comparison between dtype=float64 and str",
+                "'<' not supported between instances of 'numpy.ndarray' and 'str'",
+            ]
+        )
```

Comment:
comment that one of these is for numepxr path

Reply:
this updated error message seemed no longer needed